sql/src/test/java/io/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -169,6 +169,14 @@ public int getMaxQueryCount()
       return 1;
     }
   };
+  private static final PlannerConfig PLANNER_CONFIG_LOS_ANGELES = new PlannerConfig()
+  {
+    @Override
+    public DateTimeZone getSqlTimeZone()
+    {
+      return DateTimeZone.forID("America/Los_Angeles");
```

Comment:
Maybe use `DateTimes.inferTzfromString` will be better here.

Reply:
I'll change this